### PR TITLE
[TW-1673] VS code - share collections invite to workspaces (revert)

### PR DESCRIPTION
**Pull request by jkonrath-postman**
Reverts postmanlabs/postman-docs#5440

### CODE REVIEW

File: src/pages/docs/getting-started/basics/about-vs-code-extension.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ The VS Code extension enables you to:
 
 * Create and send HTTP, raw WebSocket, and gRPC requests.
 * Import collections and environments exported from Postman, and import requests from cURL commands.
+* Collaborate with team members by sharing collections, environments, and workspaces.
```
Comment (nerminamiller-postman) [inline src/pages/docs/getting-started/basics/about-vs-code-extension.md:34]:
```suggestion
* Collaborate with team members by sharing workspaces, environments, and collections.
```

File: src/pages/docs/collaborating-in-postman/sharing.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ You can share your collections, requests, examples, collection folders, APIs, Fl
 * Hover over the element's name in the overview page and select the link icon <img alt="Link icon" src="https://assets.postman.com/postman-docs/icon-workspace-link-v9.jpg#icon" width="18px"> to copy the element's URL to the clipboard.
 * For requests and collection folders, you can also select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the element you want to share and select **Copy link** to copy the element's URL to the clipboard.
 * To share a URL to a collection, collection folder, request, or example with an active environment selected, [choose an environment using the environment selector](/docs/sending-requests/variables/managing-environments/#switch-between-environments), copy the element's link to your clipboard, then send the link to team members you want to share with.
+* You can also use the [Postman VS Code extension](/docs/getting-started/basics/about-vs-code-extension/) to share a collection and environment, or a URL to a collection and environment.
```
Comment (nerminamiller-postman) [inline src/pages/docs/collaborating-in-postman/sharing.md:56]:
```suggestion
* You can also use the [Postman Visual Studio (VS) Code extension](/docs/getting-started/basics/about-vs-code-extension/) to share a collection and environment, or a URL to a collection and environment.
```

File: src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md [hunk 3]
```diff
@@ -125,6 +125,9 @@ You can invite team members, [groups](/docs/collaborating-in-postman/user-groups
 Your ability to share workspaces with teammates depends on your [workspace role](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles):
 
 * **Workspace Admin** - Can invite Postman teammates or external users to personal, private, team, and public workspaces, and assign them a workspace role. If a Workspace Admin invites an external user to a workspace, Postman will share this request with [Team Admins](https://learning.postman.com/docs/collaborating-in-postman/roles-and-permissions/#team-roles). Once approved, an invite to join the team will be sent to the individual.
+
+    > Workspace Admins can also use the [Postman VS Code extension](/docs/getting-started/basics/about-vs-code-extension/) to invite Postman teammates to workspaces, and assign them a workspace role.
```
Comment (nerminamiller-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md:129]:
```suggestion
    > Workspace Admins can also use the [Postman Visual Studio (VS) Code extension](/docs/getting-started/basics/about-vs-code-extension/) to invite Postman teammates to workspaces, and assign them a workspace role.
```

File: src/pages/docs/collaborating-in-postman/requesting-access-to-elements.md [hunk 4]
```diff
@@ -59,6 +59,8 @@ Team members with Admin access for the workspace or Editor access for an element
 
 1. Select **Update Roles** to save your changes.
 
+> You can also use the [Postman VS Code extension](/docs/getting-started/basics/about-vs-code-extension/) to manage collection and environment roles assigned to team members.
```
Comment (nerminamiller-postman) [inline src/pages/docs/collaborating-in-postman/requesting-access-to-elements.md:62]:
```suggestion
> You can also use the [Postman Visual Studio (VS) Code extension](/docs/getting-started/basics/about-vs-code-extension/) to manage collection and environment roles assigned to team members.
```

File: src/pages/docs/collaborating-in-postman/sharing.md (hunk 2 above)
Comment (nerminamiller-postman) [inline src/pages/docs/collaborating-in-postman/sharing.md:56]:
```suggestion
* You can also use the [Postman Visual Studio (VS) Code extension](/docs/getting-started/basics/about-vs-code-extension/) to share a collection and environment, or a URL to a collection and environment.
```

**Review by nerminamiller-postman — APPROVED** [review]
LGTM! Just some minor suggestions. Feel free to ignore!

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/collaborating-in-postman/sharing.md (hunk 2 above)
Comment (nerminamiller-postman) [reply src/pages/docs/collaborating-in-postman/sharing.md:56]: I usually refer to the product license and privacy terms. But also internal treatment of third-party names.

Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/sharing.md:56]: Thanks! I think we're OK to use "VS" on first use for now. Externally we've also been using "Postman VS Code extension" in Postman's videos and blogs.

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

**Pull request merged**